src/cs2_demo_downloader/match_scraper.py

```diff
@@ -89,34 +102,35 @@ def match_table_empty(page):
     ):
         logging.info("match_table empty")
         return True
-    logging.info("match_table not empty!")
-    return False
+    else:
```

Comment:
I can see why you put an `else` statement here, but if you think of `if rown_count == 0...` as the special case that disrupts flow here, I think the function reads better without it.

The implication being that this function will always return `False` unless something special happened.